src/_gettsim/config.py

```diff
@@ -157,7 +157,8 @@ def set_array_backend(backend: str):
     "p_id_ehepartner": int,
     "p_id_einstandspartner": int,
     "vermögen_bedürft": float,
-    "eigenbedarf_gedeckt": bool,
+    "arbeitsl_geld_2_eigenbedarf_gedeckt": bool,
```

Comment:
I don't think this one is necessary anymore if we know `wohngeld_kinderzuschl_statt_arbeitsl_geld_2`?